content/en/users/tutorials/data-transfer-grid-storage/_index.md

```diff
@@ -225,4 +231,4 @@ the `My jobs` tab shown in the following screenshot:
 
 Each line on the list shows a different job. When clicked it will expand to show
 additional details. The reason for the failure of a job which can be seen by
-moving the mouse pointer over the `File ID` on the detailed view.
+moving the mouse pointer over the `File ID` on the detailed view.
```

Comment:
Here it seems you removed the single new line character at the end of the file.

Reply:
This could be while cleaning up and adding the corrections.

Reply:
Fixed it.